### PR TITLE
Add button group support

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

Add group support for button entities.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/121715
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
  - Added support for controlling remote activities in Alexa Smart Home integration.
  - Introduced humidifier support in the Aprilaire integration.
  - Added event support for Doorbell and Motion in August integration.
  - Included Beoconnect Core device in Bang & Olufsen integration.
  - Enhanced command-line integration with `json_attributes_path`.
  - Added event entities to Doorbird and Feedreader integrations.
  - Introduced button groups in Group integration.
  - Expanded device types in Habitica integration.
  - Added valve support in Homekit and Hydrawise integrations.
  - Added new integrations for madVR and MINI Connected.
  - Enhanced MQTT integration by simplifying payload requirements.
  - Added configuration options for Netatmo public weather sensors.
  - Included keep-alive parameter in Ollama integration.
  - Introduced new entities for Roborock Zeo and Technove integrations.
  - Added sunroof cover entity in Teslemetry integration.
  - Detailed energy entities in Tessie integration.
  - Introduced event platform in UniFi Protect integration.

- **Documentation**
  - Updated ownership for Mealie integration to include new contributor.

### CODE REVIEW

**Comment by frenck** [comment]
Oops, wrong target branch. Sorry about the notification 🙏 

../Frenck

File: source/_integrations/group.markdown [hunk 1]
```diff
@@ -62,6 +63,13 @@ Binary sensor, light, and switch groups allow you set the "All entities" option.
 - Otherwise, the group state is `off` if at least one group member is `off`.
 - Otherwise, the group state is `on`.
 
+### Button groups
+
+In short, the group state is the last time the grouped button was pressed.
```
Comment (c0ffeeca7) [inline source/_integrations/group.markdown]:
```suggestion
The group state is the last time the grouped button was pressed.
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @frenck :+1:

**Pull request merged**